app/models/user.rb

```diff
@@ -66,4 +78,25 @@ def active_for_authentication?
   def inactive_message
     "Sorry, this account has been disabled."
   end
+
+  protected
+
+  # Returns whether the given user can be disabled or not. The following rules
+  # apply:
+  #   1. A user can disable himself unless it's the last admin on the system.
+  #   2. The admin user is the only one that can disable other users.
+  def can_disable?(user)
+    # The "portus" user can never be disabled.
+    return false if user.username == "portus"
```

Comment:
We already have `User.not_portus`. I think we should have something more expressive like `User#is_portus?`